### PR TITLE
Ticket 83/optimize tailwind bundle

### PR DESCRIPTION
**Pull request by namtran-kosaidovn**

### CODE REVIEW

**Comment by buinhansinh** [comment]
@namtran-kosaidovn this project is using Visual studio code, please use it to make sure there no changes in the format of unchanged file. 
After you set up tailwind, where do you apply tailwind theme? how to generate the tailwind theme? , could you update README.md

File: en/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@400;500;700&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tw-elements/dist/css/index.min.css" />
     <script src="https://cdn.tailwindcss.com"></script>
```
Comment (buinhansinh) [inline en/index.html]: We have set up tailwind for production, we don't need this script anymore , please remove all tailwind config for development

**Review by buinhansinh — CHANGES_REQUESTED** [review]

**Comment by buinhansinh** [comment]
@namtran-kosaidovn please deploy this ticket to test environment => namtran.kosaidovnit.com so that everybody can see it

File: en/index.html (hunk 1 above)
Comment (namtran-kosaidovn) [reply en/index.html]: Removed the included CDN script.

**Comment by namtran-kosaidovn** [comment]
> @namtran-kosaidovn please deploy this ticket to test environment => namtran.kosaidovnit.com so that everybody can see it

Done - http://namtran.kosaidovnit.com/vn/

**Comment by ttmthanh** [comment]
image
Please check style for signup form

**Comment by buinhansinh** [comment]
@namtran-kosaidovn  please compare changes with this original page: https://tokutei.talentasia.vn/. Please make sure there is no style difference

File: index.html [hunk 2]
```diff
@@ -18,14 +18,12 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@400;500;700&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" />
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tw-elements/dist/css/index.min.css" />
-    <script src="https://cdn.tailwindcss.com"></script>
+    <link rel="stylesheet" href="./dist/output.css" />
```
Comment (ttmthanh) [inline index.html:23]: No need `./`. Just `dist/output.css` is okay

File: en/index.html [hunk 3]
```diff
@@ -9,14 +9,13 @@
...
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@400;500;700&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tw-elements/dist/css/index.min.css" />
-    <script src="https://cdn.tailwindcss.com"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
     <script src="https://kit.fontawesome.com/b943498dde.js" crossorigin="anonymous"></script>
     <script src="src/javascripts/back-to-top.js"></script>
```
Comment (ttmthanh) [inline en/index.html:17]: Sorry. Please help me add `../src`

File: vn/index.html [hunk 4]
```diff
@@ -9,14 +9,13 @@
...
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@400;500;700&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tw-elements/dist/css/index.min.css" />
-    <script src="https://cdn.tailwindcss.com"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
     <script src="https://kit.fontawesome.com/b943498dde.js" crossorigin="anonymous"></script>
     <script src="src/javascripts/back-to-top.js"></script>
```
Comment (ttmthanh) [inline vn/index.html:17]: Sorry. Please help me add `../src`

**Comment by buinhansinh** [comment]
@ttmthanh

**Comment by buinhansinh** [comment]
This task will be completed by another task.

**Pull request closed without merging**